README.md

```diff
@@ -171,15 +171,16 @@ var ns = createCssNs({
-|-------------|--------|------------|------------
-| `namespace` | string | (none)     | Mandatory base part for the namespace, e.g. `"MyComponent"` or `__filename`. For convenience, a possible file path and suffix are ignored, so that if the provided value is `"/path/to/MyComponent.js"`, the resulting namespace will still be `"MyComponent"`.
-| `prefix`    | string | `""`       | All namespaces are prefixed with this string. By default, namespaces aren't prefixed. Class names that begin with this `prefix` are automatically excluded .
-| `include`   | regex  | `/^[a-z]/` | Only class names matching this regex are namespaced. By default, only ones starting in lower-case are. This works out nicely with upper-cased `namespace` values: it ensures only one namespace can be applied to a class name, and calling `ns()` multiple times has the same effect as calling it once.
-| `exclude`   | regex  | `/^$/`     | Class names matching this regex are not namespaced. By default, nothing is excluded (since `/^$/` won't ever match a class name). When both `include` and `exclude` match, `exclude` wins.
-| `self`      | regex  | `/^this$/` | Class names matching this regex are replaced with the name of the namespace itself. This allows you to e.g. mark the root of your UI component without any suffixes, just the component name.
-| `glue`      | string | `"-"`      | This string is used to combine the namespace and the class name.
-| `React`     | object | (none)     | Providing this option enables React integration. When provided, must be an instance of React, e.g. `{ react: require('react') }`. See [Use with React](#use-with-react) for details.
+| Option         | Type   | Default    | Description
+|----------------|--------|------------|------------
+| `namespace`    | string | (none)     | Mandatory base part for the namespace, e.g. `"MyComponent"` or `__filename`. For convenience, a possible file path and suffix are ignored, so that if the provided value is `"/path/to/MyComponent.js"`, the resulting namespace will still be `"MyComponent"`.
+| `prefix`       | string | `""`       | All namespaces are prefixed with this string. By default, namespaces aren't prefixed. Class names that begin with this `prefix` are automatically excluded .
+| `include`      | regex  | `/^[a-z]/` | Only class names matching this regex are namespaced. By default, only ones starting in lower-case are. This works out nicely with upper-cased `namespace` values: it ensures only one namespace can be applied to a class name, and calling `ns()` multiple times has the same effect as calling it once.
+| `exclude`      | regex  | `/^$/`     | Class names matching this regex are not namespaced. By default, nothing is excluded (since `/^$/` won't ever match a class name). When both `include` and `exclude` match, `exclude` wins.
+| `ignorePrefix` | string | `""`       | Class names beginning with this string are not namespaced. By default, nothing is ignored.
```

Comment:
I was thinking of the naming (my favorite topic of course!) and it'd be grand if this option would not be easily confused with ones we already have, like `exclude` and `prefix`... Would you be fine with `escape` for example? As in, you're "escaping" the prefixing altogether.

Reply:
Oh, and I think it could even have a default value. What'd be a good one? Would `=` work? As in, "I want it to be exactly this class", with `=btn`.

Reply:
Ah, yes! I was counting on you to come up with a good name for this thing :)

I think `=` would be a good default value.